attributes/config_v5.rb

```diff
@@ -0,0 +1,119 @@
+#
+#node.attr.rack: r1
+# Pre 5.0 attributes
+#default['elasticsearch']['config']['node.rack'] = nil
+default['elasticsearch']['config_v5']['node.attr.rack'] = nil
+default['elasticsearch']['config_v5']['node.attr.dc'] = nil
+default['elasticsearch']['config_v5']['node.attr.size'] = nil
+#
+# ----------------------------------- Paths ------------------------------------
+#
+# Path to directory where to store the data (separate multiple locations by comma):
+#
+#path.data: /path/to/data
+default['elasticsearch']['config_v5']['path.data'] = nil
+#
+# Path to log files:
+#
+#path.logs: /path/to/logs
+default['elasticsearch']['config_v5']['path.logs'] = '/var/log/elasticsearch'
+#
+# ----------------------------------- Memory -----------------------------------
+#
+# Lock the memory on startup:
+#
+#bootstrap.memory_lock: true
+# Pre 5.0 attributes
+#default['elasticsearch']['config']['bootstrap.mlockall'] = true
+default['elasticsearch']['config_v5']['bootstrap.memory_lock'] = true
+#
+# Make sure that the heap size is set to about half the memory available
+# on the system and that the owner of the process is allowed to use this
+# limit.
+#
+# Elasticsearch performs poorly when the system is swapping the memory.
+#
+# ---------------------------------- Network -----------------------------------
+#
+# Set the bind address to a specific IP (IPv4 or IPv6):
+#
+#network.host: 192.168.0.1
+# Pre 5.0 attributes
+#default['elasticsearch']['config']['network.bind_host'] = node['ipaddress']
+#default['elasticsearch']['config']['network.publish_host'] = node['ipaddress']
+default['elasticsearch']['config_v5']['network.host'] = node['ipaddress']
+#
+# Set a custom port for HTTP:
+#
+#http.port: 9200
+default['elasticsearch']['config_v5']['transport.tcp.port'] = 9_300
+default['elasticsearch']['config_v5']['transport.tcp.compress'] = true
+default['elasticsearch']['config_v5']['http.port'] = 9_200
+#
+# For more information, see the documentation at:
+# <http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html>
+#
+# --------------------------------- Discovery ----------------------------------
+#
+# Pass an initial list of hosts to perform discovery when new node is started:
+# The default list of hosts is ["127.0.0.1", "[::1]"]
+#
+#discovery.zen.ping.unicast.hosts: ["host1", "host2"]
+default['elasticsearch']['config_v5']['discovery.zen.ping.unicast.hosts'] = []
+#
+# Prevent the "split brain" by configuring the majority of nodes (total number of nodes / 2 + 1):
+#
+#discovery.zen.minimum_master_nodes: 3
+default['elasticsearch']['config_v5']['discovery.zen.minimum_master_nodes'] = 1
+default['elasticsearch']['config_v5']['discovery.zen.fd.ping_timeout'] = '3s'
+# Removed from ES 5.0
+#default['elasticsearch']['config_v5']['discovery.zen.ping.multicast.enabled'] = false
+#
+# For more information, see the documentation at:
+# <http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery.html>
```

Comment:
use `5.0` in doc links instead of `current` so that the correct version of the doc is always referenced and not just latest